pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -529,6 +530,7 @@ func TestScaleBounds(t *testing.T) {
 		name         string
 		min          string
 		max          string
+		config       *autoscalerconfig.Config
```

Comment:
```suggestion
		config       autoscalerconfig.Config
```

if it'd work, it'd be nicer to just use the zero value of Config